### PR TITLE
Update the docstrings

### PR DESCRIPTION
**Pull request by DrTaDa**

### CODE REVIEW

File: bluepyefe/ecode/tools.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
-"""Tools"""
+"""Tools sahred by all eCodes"""
```
Comment (anilbey) [inline bluepyefe/ecode/tools.py:1]: small typo

**Review by anilbey — APPROVED** [review]
Thanks, looks good.
If you find it informative for the documentation, the return type annotations can also be added.
E.g. 
https://github.com/BlueBrain/EModelRunner/blob/8d8e1bc2d0496d184adaa785416b22cc55f4e781/emodelrunner/synplas_analysis.py#L31

Or this way (shorter). Available since py36.
https://github.com/pandas-dev/pandas/blob/main/pandas/core/indexes/api.py#L364

**Comment by DrTaDa** [comment]
Wait wat ! Why did it close ? Noooo

**Comment by anilbey** [comment]
Can you not reopen?

**Pull request closed without merging**